source/schemes.ttl

```diff
@@ -232,6 +233,10 @@
     dc:title "Library of Congress Subject Headings"@en;
     foaf:homepage <http://id.loc.gov/authorities/subjects/> .
 
+<marc> a skos:ConceptScheme;
+    skos:notation "marc";
+    dc:title "MARC Genre Terms List"@en, "MARC Genre/Form Termlista"@sv .
```

Comment:
The `</term/marc>` scheme can't be just genre/form terms. These are all terms under `</term/marc>`.

Reply:
Actually, `</marc>`. For all "MARC terms". One pile.